o3po/includes/class-o3po-publication-type.php

```diff
@@ -2069,7 +2074,9 @@ protected function the_admin_panel_doi( $post_id ) {
 		echo '	<tr>';
 		echo '		<th><label for="' . $post_type . '_doi" class="' . $post_type . '_doi_label">' . 'Doi' . '</label></th>';
 		echo '		<td>';
-		echo '			<input type="text" readonly value="' . esc_attr($doi) . '" ><br /><p>(The doi is automatically calculated from the above meta data and is of the form ' . $this->get_journal_property('journal_level_doi_suffix') . '-YYYY-MM-DD-pages. If not enough information is available, it is not set and the post is forced to private)</p>';
+		echo '			<input type="text" readonly value="' . esc_attr($doi) . '" ><br /><p>(The doi is automatically calculated from the above meta data and is of the form ' 
+                     . str_replace( '[journal_level_doi_suffix]', $this->get_journal_property('journal_level_doi_suffix'), $this->get_journal_property('doi_suffix_template') ) 
+                     . '. If not enough information is available, it is not set and the post is forced to private)</p>';
```

Comment:
Maybe it would be nice to add a link to the settings page where this can be configured here? E.g., something like "The DOI format can be configured in the settings", with "settings a link to the respective settings page?

Reply:
Do you mean the `journal level DOI`? Its right above the `DOI template`. [This is how it looks](https://imgur.com/a/9hZtHGo). Is the `journal level DOI` used in another context? If not, it could be just a part of the template.

Reply:
I meant a link to the journal settings page. I have implemented this in 1500ffc